frontend/app/routes/$lang+/_protected+/letters+/$id.download.tsx

```diff
@@ -30,7 +46,7 @@ export async function loader({ context: { session }, params, request }: LoaderFu
     headers: {
       'Content-Type': 'application/pdf',
       'Content-Length': decodedPdfBytes.length.toString(),
-      'Content-Disposition': `inline; filename="${params.id}.pdf"`,
+      'Content-Disposition': `inline; filename="${documentName}.pdf"`,
```

Comment:
You should sanitize this in case the filename contains an illegal character (ie: `/` or `?` or many more).

Reply:
will do it in my next PR